### PR TITLE
Adm 385 add multiple sets of mock data with different pipeline ids for metrics page.

### PR DESCRIPTION
**Pull request by leeranzhi**
## Summary
- add multiple sets of mock data with different pipeline ids for metrics page.
...

## Before

_Description_

Before

## After

_Description_

After

## Note

_Null_

### CODE REVIEW

File: stubs/backend/buildkite/buildkite-stubs.yaml [hunk 1]
```diff
@@ -26,14 +26,14 @@
 # Pipeline builds
 - request:
     method: GET
-    url: /v2/organizations/XXXX/pipelines/(.*)/builds
+    url: /v2/organizations/XXXX/pipelines/(fs-[a-zA-Z-]+)/builds
     query:
-      page: 1
+      page: "(\\d+)"
```
Comment (allenzph) [inline stubs/backend/buildkite/buildkite-stubs.yaml:31]: this pr has too many file changes, is there something wrong?

Comment (leeranzhi) [reply stubs/backend/buildkite/buildkite-stubs.yaml:31]: I have rebase the brach again, now it's clearly.

File: stubs/backend/buildkite/buildkite-stubs.yaml [hunk 2]
```diff
@@ -26,14 +26,14 @@
 # Pipeline builds
 - request:
     method: GET
-    url: /v2/organizations/XXXX/pipelines/(.*)/builds
+    url: /v2/organizations/XXXX/pipelines/(fs-[a-zA-Z-]+)/builds
```
Comment (guzhongren) [inline stubs/backend/buildkite/buildkite-stubs.yaml:29]: Replace XXXX with a regex

Comment (leeranzhi) [reply stubs/backend/buildkite/buildkite-stubs.yaml:29]: Fixed.

**Pull request merged**